sw/device/lib/testing/keymgr_testutils.h

```diff
@@ -26,7 +26,7 @@ static const dif_keymgr_versioned_key_params_t kKeyVersionedParams = {
             0x322288d8,
             0xde919d54,
         },
-    .version = 0xaa,
+    .version = 0x11,
```

Comment:
@jadephilipoom I simply changed this to a smaller value, so that we can directly use this param in both Creator and OwnerInt states.

Reply:
do you need me to help take a look? it seems odd at first sigh the version would cause a failure unless the test is trying to supply a bigger version than expected.

Reply:
I changed it to `0x1` in #14677, but `0x11` is probably better.

Reply:
> do you need me to help take a look? it seems odd at first sigh the version would cause a failure unless the test is trying to supply a bigger version than expected.

@tjaychen after fixing the issue in dif_keymgr, this version indeed caused a failure, so lower it to 0x11 since we set max version to 0x11 for Creator state

Reply:
o i see what you're saying. I interpreted it the other way around and thought there was a keymgr bug.
Thanks for explaining!